### PR TITLE
do not put inherit block in if statement

### PR DESCRIPTION
**Pull request by benrudolph**
@dannyroberts @biyeun this more closely matches how we do this in `style/base.html` don't know if it's the fix. i was only able to test compress locally (not staging)

### CODE REVIEW

File: corehq/apps/cloudcare/templates/landing_page/base.html [hunk 1]
```diff
@@ -3,18 +3,23 @@
...
 <link type="text/less"
       rel="stylesheet"
       media="all"
-      href="{% static 'cloudcare/less/formplayer-webapp.debug.less' %}" />
-{% endblock %}
+      href="{% static 'cloudcare/less/formplayer-webapp.debug.less' %}"/>
 
-{% block css %}
+{% else %}
+{% compress css %}
```
Comment (benrudolph) [inline corehq/apps/cloudcare/templates/landing_page/base.html]: this page is only behind a feature flag that no one uses, but it was missing a compress tag

**Comment by dannyroberts** [comment]
I guess this is worth a shot? If we find out what's causing it, we probably also want to make whatever thing causes a compress block not to be run fail really loudly.

**Review by dannyroberts — APPROVED** [review]

**Comment by biyeun** [comment]
hmmmm.... did this actually fix the issue?

**Comment by dannyroberts** [comment]
looks like it!

**Comment by benrudolph** [comment]
magic...

**Pull request merged**